lib/Domain/TextEdit.php

```diff
@@ -36,9 +36,9 @@ public static function applyEdits(array $edits, string $text) : string
         for ($i = \count($edits) - 1; $i >= 0; $i--) {
             $edit = $edits[$i];
 
-            if ($prevEditStart < $edit->start) {
+            if ($prevEditStart < $edit->start || $prevEditStart < $edit->start + $edit->length) {
```

Comment:
This mean that either `$edit->start` or `$edit->length` can be null ?
Kind of strange, should it even be possible ?